01_AppCore/Business/Models/Results/ExceptionResult.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+namespace _01_AppCore.Business.Models.Results
+{
+    public class ExceptionResult : Result
+    {
+        public ExceptionResult(Exception exception, bool showException = true)
+            : base(ResultStatus.Exception,
+                  showException ?
+                    (exception != null ?
```

Comment:
There can be even more exceptions here. I'd go over them in a loop add their messages together.